src/retry.ts

```diff
@@ -1,6 +1,8 @@
+import { env } from 'node:process'
+
 import type { Fetcher } from './types.ts'
 
-const DEFAULT_RETRY_DELAY = 5000
+const DEFAULT_RETRY_DELAY = env.NODE_ENV === 'test' ? 1 : 5000
```

Comment:
Speeds up tests in the CI, as we don't have to wait a few seconds when a test is about to fail.